spr/src/commands/init.rs

```diff
@@ -74,7 +74,27 @@ pub async fn init() -> Result<()> {
         return Err(Error::new("Cannot continue without an access token."));
     }
 
+    let github_api_domain = dialoguer::Input::<String>::new()
+        .with_prompt("Github API domain (override for Github Enterprise)")
+        .with_initial_text(
+            config
+                .get_string("spr.githubApiDomain")
+                .ok()
+                .unwrap_or_else(|| "api.github.com".to_string()),
+        )
+        .interact_text()?;
+
+    config.set_str("spr.githubApiDomain", &github_api_domain)?;
+
+    let api_base_url;
+    if github_api_domain == "api.github.com" {
+        api_base_url = "https://api.github.com/v3/".into()
```

Comment:
This should be without the `/v3/` to make it work for me.

Reply:
@andrewhamon can you check if this is correct? I haven't tested this PR personally yet, but it looks like this is the non-enterprise case, so we obviously need to not break that too :)